`write_vtk_lagrange` for order 0

bmxam opened the pull request:
Fix for `write_vtk_lagrange` for a Lagrange P0.

Closes #113 .

ghislainb commented on src/writers/vtk.jl:
```diff
@@ -372,19 +372,34 @@ function write_vtk_lagrange(
+        vals = map(values(vars)) do var
+            _, dim = get_return_type_and_codim(var, mesh)
+            is_scalar = dim[1] == 1
+            value = var_on_centers(var, mesh)
```
Why not the `cell_mean` ?

ghislainb reviewed (commented):
`U_export` is supposed to be the order of the mesh up to now. I think we should refine the API during a live discussion before merging this PR. Order 0 are currently handled with `MeshCellData` computed with the function `cell_mean()` before calling `write_vtk_lagrange`

bmxam commented:
@ghislainb I have kept the actual function signature (with `U_export`) and added a new one (with `mesh_degree` etc). I have also added the `cell_mean` of a `MeshData{CellData}`.
